src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -303,9 +303,6 @@
     <value>EnumConverter is not yet supported on .NET Standard 2.0.</value>
   </data>
   <data name="FormatDateTime" xml:space="preserve">
-    <value>The JSON value is of unsupported or incorrect ISO 8601 format.</value>
-  </data>
-  <data name="FormatDateTimeOffset" xml:space="preserve">
-    <value>The JSON value is of unsupported or incorrect ISO 8601 format.</value>
+    <value>The JSON value is of unsupported DateTime format.</value>
```

Comment:
Since we are specifying DateTime here, we could add another one with DateTimeOffset.

For example, we have, above:
- The JSON value is either too large or too small for a Decimal.
- The JSON value is either too large or too small for a Double.